src/domains/ecommerce-common/00_alerts.tf

```
@@ -298,4 +298,48 @@ AzureDiagnostics
     operator  = "GreaterThanOrEqual"
     threshold = 2
   }
-}
\ No newline at end of file
+}
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_notifications_service_availability" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "ecommerce-notifications-service-availability-alert"
+  resource_group_name = azurerm_resource_group.rg_ecommerce_alerts[0].name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id, azurerm_monitor_action_group.ecommerce_opsgenie[0].id]
+    email_subject          = "[eCommerce] Availability Alert"
+    custom_webhook_payload = "{}"
+  }
+  data_source_id = data.azurerm_api_management.apim.id
+  description    = "eCommerce notifications Availability less than or equal 99%"
+  enabled        = true
+  query = (<<-QUERY
+let thresholdTrafficMin = 150;
+let thresholdTrafficLinear = 400;
+let lowTrafficAvailability = 96;
+let highTrafficAvailability = 99;
+let thresholdDelta = thresholdTrafficLinear - thresholdTrafficMin;
+let availabilityDelta = highTrafficAvailability - lowTrafficAvailability;
+AzureDiagnostics
+| where url_s == 'https://api.platform.pagopa.it/ecommerce/notifications-service/v1/emails'
+| summarize
+    Total=count(),
+    Success=countif(responseCode_d < 500 or url_s startswith "https://api.platform.pagopa.it/ecommerce/notifications-service/v1/emails" and ( responseCode_d == 502 or responseCode_d == 504))
```

```suggestion
    Success=countif(responseCode_d < 500)
```

I would count as successful requests all request except 5xx ones. what do you think?

---

Done here 9bf9bffd6904ae5e92dfdb9c55d43ee4f1d110ff